Userland/Applets/Keymap/KeymapStatusWidget.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibCore/FileWatcher.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/Menu.h>
+#include <LibGUI/Window.h>
+
+class KeymapStatusWidget : public GUI::Label {
+    C_OBJECT(KeymapStatusWidget);
+
+    virtual void mousedown_event(GUI::MouseEvent& event) override;
+
+    void set_current_keymap(String const& keymap, bool clear_background = true);
+
+private:
+    KeymapStatusWidget();
+
+    RefPtr<GUI::Menu> m_context_menu;
+    String m_current_keymap;
+
+    ErrorOr<void> refresh_menu();
+
+    char const* m_keyboard_config = "/etc/Keyboard.ini";
```

Comment:
This doesn't need to be a member, it never changes. Just hardcode it where it's actually being used (the same way you hardcoded `"/bin/KeyboardSettings"`).

Also should be a StringView instead of a C string otherwise.